src/system.ts

```diff
@@ -37,6 +53,11 @@ export const bootstrapCache = async () => {
   if (PREFETCH_CONTRACTS) {
     await prefetchContracts();
   }
+
+  if (SAVE_CACHE_TO_S3) {
+    // save the cache on a 3 hour interval
```

Comment:
nit: comment is dated. could be removed as its kind duplicating what the code it saying above and below `SAVE_CACHE_TO_S3` , `setInterval`

Reply:
```suggestion
```